src/test/java/core/basesyntax/service/CsvFileReaderImplTest.java

```diff
@@ -0,0 +1,33 @@
+package core.basesyntax.service;
+
+import org.junit.Assert;
+import org.junit.BeforeClass;
+import org.junit.Test;
+
+public class CsvFileReaderImplTest {
+    private static final String validPathToFile
+            = "src/main/java/core/basesyntax/files/activities.csv";
+    private static final String invalidPathToFile
+            = "src/main/java/core/basesyntax/activities.csv";
+    private static final String InvalidPathMessage
+            = "Can't read from path: ";
+    private static CsvFileReader csvFileReader;
+
+    @BeforeClass
+    public static void setUp() {
+        csvFileReader = new CsvFileReaderImpl();
+    }
+
+    @Test
+    public void readWorkabilityOk() {
+        csvFileReader.read(validPathToFile);
```

Comment:
what is this test? we need to check if it's reading ok

Reply:
I'm speechless

Reply:
At that time I thought If there are no exceptions, then there are no problems :D